docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+
+services:
+  sirbot:
+    image: pyslackers/sirbot:latest
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file:
+      - .env
+    restart: unless-stopped
+    expose:
+      - "8080"
```

Comment:
If you are wanting to expose to the host, you actually want to use `ports: - "external:internal"`